README.md

```diff
@@ -87,93 +87,93 @@
+- ![](https://img.shields.io/github/stars/feednext/feednext.svg?style=flat-square) [Feednext](https://github.com/feednext/feednext) - Open-Source Social Media Application.
+- ![](https://img.shields.io/github/stars/ever-co/gauzy.svg?style=flat-square) [Gauzy](https://github.com/ever-co/gauzy) - Open-Source Profits Sharing Platform for modern agencies and studios.
+- ![](https://img.shields.io/github/stars/notadd/notadd.svg?style=flat-square) [Notadd](https://github.com/notadd/notadd) - Microservice development architecture.
+- ![](https://img.shields.io/github/stars/novuhq/novu.svg?style=flat-square) [Novu](https://github.com/novuhq/novu) - The open-source notification infrastructure with fully functional embedded notification center.
+- ![](https://img.shields.io/github/stars/valueadd-poland/pimp-my-pr.svg?style=flat-square) [Pimp My PR](https://github.com/valueadd-poland/pimp-my-pr) - Open-Source platform for statistics and pull request management.
+- ![](https://img.shields.io/github/stars/teableio/teable.svg?style=flat-square) [Teable](https://github.com/teableio/teable) - A Super fast, Real-time, Professional, Developer-friendly, No code database.
+- ![](https://img.shields.io/github/stars/ToolJet/ToolJet.svg?style=flat-square) [ToolJet](https://tooljet.io/) - ToolJet is the open-source low-code framework alternative to Retool & Mendix to build & deploy internal tools with minimal engineering effort. ([Source Code](https://github.com/ToolJet/ToolJet)) `GPL-3.0`
+- ![](https://img.shields.io/github/stars/vendure-ecommerce/vendure.svg?style=flat-square) [Vendure](https://github.com/vendure-ecommerce/vendure) - Open-Source headless GraphQL ecommerce framework built on NestJS, with a focus on developer productivity and ease of customization.
+- ![](https://img.shields.io/github/stars/pvarentsov/iola.svg?style=flat-square) [iola](https://github.com/pvarentsov/iola) - Socket client with Rest API.
+- ![](https://img.shields.io/github/stars/amplication/amplication.svg?style=flat-square) [Amplication](https://github.com/amplication/amplication) - Amplication is an open-source low-code devtool that auto-generates backend apps built with TypeScript and Node.js, and a client built with React.
+- ![](https://img.shields.io/github/stars/necordjs/toolkit.svg?style=flat-square) [Necord/Toolkit](https://github.com/necordjs/toolkit) - Toolkit is an open-source bot for searching and sending documentation, built with Necord.
+- ![](https://img.shields.io/github/stars/undb-xyz/undb.svg?style=flat-square) [Undb](https://github.com/undb-xyz/undb) - Undb is a no-code database based on Svelte-kit and SQLite.
 
 ## Components & Libraries
 
 #### Utilities
 
-- [`@nestjs/cqrs`](https://github.com/nestjs/cqrs) - A lightweight CQRS module for Nest framework.
+- ![](https://img.shields.io/github/stars/nestjs/cqrs.svg?style=flat-square) [`@nestjs/cqrs`](https://github.com/nestjs/cqrs) - A lightweight CQRS module for Nest framework.
```

Comment:
I don't think we should do the same for those packages that are in the nestjs org because the stars are supposed to be used to help people on choosing which community package they could use

Reply:
makes sense, fixed it